kernel/kernel-auth-service/src/main/java/io/mosip/kernel/auth/service/impl/AuthServiceImpl.java

```diff
@@ -537,23 +494,6 @@ public UserDetailsResponseDto getUserDetailBasedOnUserId(String appId, List<Stri
 
 	@Override
 	public MosipUserDto valdiateToken(String token) {
```

Comment:
We are only checking for an empty token. Then why are we naming this method as validateToken. Its not validating anything.

Reply:
Hi Sasi,
That method is not even used by our controller that is code of custom IAM . I am currently reviewing and code cleanup is also done by our team after which it will be removed.I Just removed it from here from find and replace that's why change is shown here

Reply:
This method is not even used by our controller . This is code of old IAM which was there as a part of migration strategy which will be removed after this  version. Our team is working on code cleanup which will be done in next build.